lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -660,8 +633,50 @@ class CkParagraphBuilder implements ui.ParagraphBuilder {
     return properties;
   }
 
+  /// Determines if the given [text] contains any code points which are not
+  /// supported by the current set of fonts.
+  void _verifyFontsSupportText(String text) {
+    CkTextStyle style = _peekStyle();
+    List<String> fontFamilies =
+        _getActualFontFamilies(style.fontFamily, style.fontFamilyFallback);
+    List<SkTypeface> typefaces = <SkTypeface>[];
+    for (var font in fontFamilies) {
+      List<SkTypeface>? typefacesForFamily =
+          skiaFontCollection.familyToTypefaceMap[font];
+      if (typefacesForFamily != null) {
+        typefaces.addAll(typefacesForFamily);
+      }
+    }
+    List<int> codeUnits = text.codeUnits;
```

Comment:
Can we avoid creating a list by using https://api.flutter.dev/flutter/dart-core/String/codeUnitAt.html instead? (I'm not sure what `codeUnitAt` does internally, but it might be more efficient.

Reply:
Done.